doc/content/sensor-edition/beta-welcome/_index.md

```diff
@@ -0,0 +1,60 @@
+---
+title: "WELCOME, WE ARE EXCITED!"
+description: ""
+weight: -1
+distributions: null
+---
+
+If you are reading this, you are probably one of the first people to get their hands on the {{% gnse %}}.
+We are extremely excited to have you as part of the community.
+
+<!--more-->
+
+The {{% gnse %}} is a unique device for multiple reasons:
+- It is a functional device that can be used for many LoRaWAN applications
+- It is a production ready device that incorporates ST new STM32WL SOC
```

Comment:
`production-ready`

Reply:
also, I think "new STMicroelectronics STM32WL SOC" or "STMicroelectronics' new STM32WL SOC" is correct

Reply:
Done